frontend/src/pages/NewStudent.vue

```diff
@@ -133,27 +136,55 @@ export default {
   },
 
   computed: {
-    filteredLanguageDataArray() {
-        return this.languageData.filter((option) => {
-            return option
-              .toString()
-              .toLowerCase()
-              .indexOf(this.nativeLanguage.toLowerCase()) >= 0
-            })
-    },
-      filteredSourceDataArray() {
+    filteredSourceDataArray() {
         return this.sourceData.filter((sourceOption) => {
             return sourceOption
               .toString()
               .toLowerCase()
               .indexOf(this.source.toLowerCase()) >= 0
             })
-    }  
+    },
+    
+  },
+  mounted() {
+    // Call API for Native Languages and add to array API_nativeLanguage
+    fetch("/api/nativeLanguages")
+      .then(response => response.json())
+      .then(result => this.API_nativeLanguage = result)
   },
-
 
   methods: {
+    filteredLanguageDataArray(language = "") {
+      this.languages = this.API_nativeLanguage.filter((option) => {
+          return option.NativeLanguage
+            .toLowerCase()
+            .includes(language || "".toLowerCase())
+          
+          })
+
+    },
     createStudent(){
+      const studentCreate = {  
+        method: "POST",
+        headers: {
+          'Accept': 'application/json',
+          'Content-Type': 'application/json'
+        },
+        body: JSON.stringify({
+          PhoneNumber: this.PhoneNumber,
+          // Replace FirstName and LastName with Name when Student Model is changed
+          FirstName: this.name,
+          LastName: 'Hossein', // Placeholder 
+          Source: this.source,
+          NativeLanguageID: this.selected.NativeLanguageID,
```

Comment:
@lauratsui is working on a [task](https://bootcamp-irrc.atlassian.net/browse/IRRC-66) that will allow the backend to just accept the native language as a string, which should make this part simpler.

Reply:
Let's call it `NativeLanguageString` for consistency with `StatusString`.